pym/_emerge/actions.py

```diff
@@ -1646,6 +1646,13 @@ def action_info(settings, trees, myopts, myfiles):
 		last_sync = portage.grabfile(os.path.join(repo.location, "metadata", "timestamp.chk"))
 		if last_sync:
 			append("Timestamp of repository %s: %s" % (repo.name, last_sync[0]))
+		last_sync = portage.grabfile(os.path.join(repo.location, "metadata", "timestamp.commit"))
+		if last_sync:
+			try:
+				head_commit = last_sync[0].split()[0]
+				append("Head commit of repository %s: %s" % (repo.name, head_commit))
+			except IndexError:
+				pass
```

Comment:
In order to avoid swallowing unintended exceptions, I prefer to limit the scope of the exception handler as much as possible, for example:
```
	try:
		head_commit = last_sync[0].split()[0]
 	except IndexError:
		pass
	else:
		append("Head commit of repository %s: %s" % (repo.name, head_commit))
```
This also makes it easier for the reader to discern where the exceptions are expected to be raised from.